content/developer/reference/backend/testing.rst

```diff
@@ -751,6 +751,27 @@ run inside it.
   - only works locally
   - only works if the test / tour can run correctly locally
 
+``debug=True``
+**************
+
+When running a tour locally via the test suite, the ``debug=True``
+parameter can be added to the ``browser_js`` or ``start_tour``
+call::
+
+    self.start_tour("/web", code, debug=True)
+
+This will automatically open a Chrome window with the tour being
+run inside it in fullscreen, with a devtools and with a debugger at the
+starting of the tour. This runs the tour with debug=assets query parameter.
+When an error throws, the browser stays opened and triggers a debugger.
```

Comment:
This will automatically open a fullscreen Chrome window with opened devtools and a debugger breakpoint set at the start of the tour. The tour is ran with the debug=assets query parameter.
When an error is thrown, the debugger stops on the exception.